client/src/state/history/readOneBlockQuery.js

```diff
@@ -7,36 +7,31 @@ import gql from 'graphql-tag';
 const query = gql`
 query ReadHistoryViewerBlock ($block_id: ID!, $limit: Int!, $offset: Int!) {
   readOneBlock(
-    Versioning: {
-      Mode: LATEST
+    versioning: {
+      mode: LATEST
     },
-    ID: $block_id
+    id: $block_id
```

Comment:
This complains when using with GraphQL v4:

```
query ReadHistoryViewerBlock($block_id: ID!, $limit: Int!, $offset: Int!) {↵  readOneBlock(versioning: {mode: LATEST}, id: $block_id) {...
```

```
{"data":null,"errors":[{"message":"Unknown argument \"id\" on field \"readOneBlock\" of type \"Query\".","code":0,
```

Reply:
Have resolved this.